### PR TITLE
Adding permissions for pending affidavit review account

### PR DESCRIPTION
**Pull request by sumesh-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/5860

*Description of changes:*
- Adding permissions for pending affidavit review account

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/routes/index.ts [hunk 1]
```diff
@@ -116,9 +116,6 @@ router.beforeEach((to, from, next) => {
         } else {
           return next({ path: `/${Pages.ACCOUNT_FREEZE}` })
         }
-      } else if (currentOrganization?.statusCode === AccountStatus.PENDING_AFFIDAVIT_REVIEW) {
```
Comment (sumesh-aot) [inline auth-web/src/routes/index.ts:119]: Moved this logic to common components.

File: docs/docs/api_contract/auth-api-1.0.0.yaml [hunk 2]
```diff
@@ -3245,6 +3245,7 @@ paths:
                     roles:
```
Comment (sumesh-aot) [inline docs/docs/api_contract/auth-api-1.0.0.yaml:3245]: Updating mock response here to make sure the payment changes works

File: auth-web/src/routes/index.ts (hunk 1 above)
Comment (saravanpa-aot) [reply auth-web/src/routes/index.ts:119]: this might still be needed for with in app navigations...

**Review by saravanpa-aot — APPROVED** [review]
please check the comment , otherwise looks okay

File: auth-web/src/routes/index.ts (hunk 1 above)
Comment (sumesh-aot) [reply auth-web/src/routes/index.ts]: Good catch! added it back.

**Pull request merged**